<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11392" started-at="2014-12-29T12:58:29Z" finished-at="2014-12-29T12:58:40Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11392" started-at="2014-12-29T12:58:29Z" finished-at="2014-12-29T12:58:40Z">
      <class name="com.example.tests.AddressRemoveTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.AddressRemoveTest@2d6d8735]" name="setUp" is-config="true" duration-ms="8200" started-at="2014-12-29T15:58:29Z" finished-at="2014-12-29T15:58:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="deleteAddress(com.example.tests.AddressDate)[pri:0, instance:com.example.tests.AddressRemoveTest@2d6d8735]" name="deleteAddress" duration-ms="2588" started-at="2014-12-29T15:58:37Z" data-provider="randomValidAddressGenerator" finished-at="2014-12-29T15:58:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[AddressDate [lastname= ,fname= ,  address= , home= , mobile=test2114706880, work=test552023778, email= , email2=test784109097, bday=1, bmonth=January, byear=1980, groupdate=group1, address2=test-296683492, phone2=test-746103443]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Lists differ at element [1]: AddressDate [lastname=,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=] != AddressDate [lastname=Select all,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=] expected [AddressDate [lastname=,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=]] but found [AddressDate [lastname=Select all,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Lists differ at element [1]: AddressDate [lastname=,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=] != AddressDate [lastname=Select all,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=] expected [AddressDate [lastname=,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=]] but found [AddressDate [lastname=Select all,fname=,  address=, home=, mobile=, work=, email=, email2=, bday=, bmonth=, byear=, groupdate=, address2=, phone2=]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:549)
	at org.testng.Assert.assertEquals(Assert.java:514)
	at com.example.tests.AddressRemoveTest.deleteAddress(AddressRemoveTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteAddress -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.AddressRemoveTest@2d6d8735]" name="tearDown" is-config="true" duration-ms="585" started-at="2014-12-29T15:58:40Z" finished-at="2014-12-29T15:58:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.AddressRemoveTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
